resources/views/home-layout.blade.php

```diff
@@ -0,0 +1,61 @@
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <script src="/dist/semantic.min.js"></script>
```

Comment:
this will be change to `/build/js/app.js` and either for css in the mix library I have added both bootstrap and semantic ui